serverless.yml

```diff
@@ -441,6 +443,35 @@ resources:
           - AttributeName: 'id'
             AttributeType: 'S'
 
+    governanceProposalsData:
+      Type: AWS::DynamoDB::Table
+      Properties:
+        TableName: ${self:custom.table.GOVERNANCE_PROPOSALS_DATA}
+        BillingMode: PAY_PER_REQUEST
+        KeySchema:
```

Comment:
did we not want network as part of the key?

Reply:
we have it in `GlobalSecondaryIndexes`, u can have only 1 primary index in ddb

Reply:
oh i'm blind